src/components/dropdowns/CynDropdown.vue

```diff
@@ -1,40 +1,34 @@
 <template>
-    <div id="dropdown-div">
-        <label for="users">Select Users:</label>
-        <br />
-        <select v-model="selected">
-            <option value disabled>Please Select a User</option>
-            <option v-bind:value="user.name" v-for="user in users" :key="user.id"
-                >{{ user.name }} | ID: {{ user.id }}</option
-            >
-        </select>
-        <p>Current user selected: {{ selected }}</p>
-    </div>
+  <div id="dropdown-div">
+    <label for="users">Select an Item:</label>
+    <br />
+    <select @change="onChange($event)">
```

Comment:
Like we talked about in slack:

Refactor this and the corresponding `onChange()` method on line 25 and use vue's inline event handling on the `<option>` on line 7 i.e. `<option v-on:click="selectedItem = item" ... > ... `

We want to use vue's helper methods whenever we can. Yes,  `@change` is a vue helper, but by using the click event we are doing a few things: 
- consolidating the logic into the helper attribute on the element
- removing a listener on the value change event (listeners are typically costly)